week-3/Homework/mandatory/7-password-validator.js

```diff
@@ -1,69 +1,68 @@
-/* 
-Password Validation
-
-Write a program that should check if each password in an array 
-contains a valid password (see below for password criterias) and return 
-new array with true or false booleans.
-
-Passwords must 
-- Have at least 5 characters.
-- Have English uppercase letters (A-Z)
-- Have English lowercase letters (a-z)
-- Have numbers (0-9)
-- Have non-alphanumeric symbols ("!", "#", "$", "%", ".")
-
-Passwords must not be any previous password in the passwords array. 
-
-Example 1:
-PreviousPassword = ["fhD8!yrjj", "ttkTu.wer3", "dvyyeyY!5", "qwbfj76%", "tytT3729."];
-
-Expected Result:
-PasswordValidationResult=  [false, false, false, false, true]
-
-*/
+// this function does validate any password within the password array
+function validatePasswordSimple(password) {
```

Comment:
This is really nice!

Reply:
Thanks, Chris, all this is due to the hard work of the mentors and course leads.